File: official/guides/customs-guide/csharp/create-customs-info.cs
CustomsInfo customsInfo = await CustomsInfo.Create(new Dictionary<string, object>() {
  { "customs_certify", true }, { "eel_pfc", "NOEEI 30.37(a)" },
  { "customs_signer", "Jarrett Streebin" }, { "contents_type", "gift" },
  { "customs_items", new List<Dictionary<string, object>>() { customsItem1, customsItem2 } }
});

File: official/guides/customs-guide/python/create-customs-info.py
import easypost


customs_info = easypost.CustomsInfo.create(
    eel_pfc="NOEEI 30.37(a)",
    customs_certify=True,
    customs_signer="Jarrett Streebin",
    contents_type="gift",
    customs_items=[customs_item1, customs_item2],
)
